hawc/apps/riskofbias/migrations/0026_fix_blank_text.py

```diff
@@ -0,0 +1,18 @@
+from django.db import migrations
+
+from ..constants import NR_SCORES
+
+
+def forwards_func(apps, schema_editor):
+    RiskOfBiasScore = apps.get_model("riskofbias", "RiskOfBiasScore")
+    RiskOfBiasScore.objects.filter(score__in=NR_SCORES, notes="<p> </p>").update(notes="<p>-</p>")
```

Comment:
prior to #638, on the server this was considered valid b/c empty text was valid. We migrate the blank to a hyphen, so that reviews previously considered final are still considered final